core/src/test/java/cucumber/runtime/formatter/FormatterBuilder.java

```diff
@@ -1,8 +1,10 @@
 package cucumber.runtime.formatter;
 
+
```

Comment:
This file only has whitespace changes.

Reply:
I updated this file to be the same from the master. It still shows a diff as a bracket position change. M not sure how to remove that diff if thats what was needed.

Reply:
The old file ended with a new line. That's the only difference left now.

Reply:
Updated :)

sorry it took this long to get this resolved.